addons/im_livechat/static/src/core/common/chat_window_patch.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="mail.ChatWindow" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-mail-ChatWindow')]" position="attributes">
+            <attribute name="t-ref">chatWindow</attribute>
+        </xpath>
```

Comment:
Would put `t-ref="root"` on the original template, for idiomatic naming of top-level root of many of similar components.

Update: this is not necessary as required LOCs should be removed too (see 2nd comment)